easybuild/easyconfigs/r/RELION/RELION-3.0.7-foss-2019a.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '3.0.7'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['https://github.com/3dem/relion/archive']
+sources = [
+    '%(version)s.tar.gz',
+]
+patches = [
+    '%(name)s-3.0.4_no_which_cmd.patch',
+    '%(name)s-3.0.4_drop_usage_of_mpi_include_path.patch',
+    '%(name)s-3.0.5_add_hpc2n_submit_file_args.patch',  # This patch is a site specific patch. Either adapt or drop.
+    ('relion_sbatch.sh', '.'),  # This submit file template is used by the hpc2n_submit_file patch.
+]
+checksums = [
+    '385ae3ae992cef60aa4ce97075dd4580bf00172154af426adf5ec509e248daac',  # 3.0.7.tar.gz
+    'bfdaf1ce856d20c27a5fcf50252394e81d08259899abc3564f6c7bcc2b4000de',  # RELION-3.0.4_no_which_cmd.patch
+    # RELION-3.0.4_drop_usage_of_mpi_include_path.patch
+    'c135b72ff8d4c3015a3b293fa07a2283396bfb3fd0658d5e938ec33fca4dcebc',
+    '054e382e19f7bfd5f016017189e4132337f7de4dc6bf2996f5933bfade61b2f4',  # RELION-3.0.5_add_hpc2n_submit_file_args.patch
+    '9305f869a07a1beaaa7e3a4670625a045c0cc4762b6b542039123e5a2731023d',  # relion_sbatch.sh
+]
+
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [
```

Comment:
Why not also add the FFTW dependency ?

Reply:
FFTW is implicit due to foss(cuda) toolchain, so one should not add it here.